README.md

```diff
@@ -54,6 +54,32 @@ Starting from zone.js v0.8.9, you can choose which web API module you want to pa
 For more details, please
 see [MODULE.md](MODULE.md).
 
+## Bundles
```

Comment:
I think this would be easier to read if it was formatted as two tables. One for bundles and one for extra patches.

Reply:
Sure, will do!